exporter/collector/integrationtest/testdata/fixtures/metrics/untyped_prometheus_metrics_expect.json

```diff
@@ -0,0 +1,1234 @@
+{
+  "createTimeSeriesRequests": [
+    {
+      "name": "projects/fakeprojectid",
+      "timeSeries": [
+        {
+          "metric": {
+            "type": "workload.googleapis.com/scrape_series_added",
+            "labels": {
+              "service_instance_id": "localhost:2222",
+              "service_name": "demo"
+            }
+          },
+          "resource": {
+            "type": "generic_task",
+            "labels": {
+              "job": "demo",
+              "location": "global",
+              "namespace": "",
+              "task_id": "localhost:2222"
+            }
+          },
+          "metricKind": "GAUGE",
+          "valueType": "DOUBLE",
+          "points": [
+            {
+              "interval": {
+                "endTime": "1970-01-01T00:00:00Z"
+              },
+              "value": {
+                "doubleValue": 22
+              }
+            }
+          ]
+        },
+        {
+          "metric": {
+            "type": "workload.googleapis.com/fake_untyped_metric",
+            "labels": {
+              "ex_com_lemons_untyped": "13",
+              "prometheus_untyped_metric": "true",
```

Comment:
this is the fixture for testing the featuregate. this label would be dropped in the gmp exporter if the gate was enabled